ko-search/movies-evan-likes/src/App/components/MovieList/MovieItem.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { centeredContainer } from 'sharedStyles.css';
+import { container, formatMovie, title, movieTitle, text } from './styles.css';
+
+export default props =>
+  <div className={(centeredContainer, formatMovie)} onClick={props.showEvanThoughts}>
+    <div className={(text, title)}>
+      <a onClick={props.handleTitleClick} className={movieTitle} href={props.movie.url}>{`${props
+        .movie.title}  | `}</a>
+      <strong>
+        {'Rating '}
+        {props.movie.rating}
+      </strong>
+      {' | '}
```

Comment:
Honestly, I use prettier, and it is pretty opinionated in it's styling choices, so it decided that for me.